packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -269,7 +269,7 @@ module.exports = {
                 options: {
                   importLoaders: 1,
                   modules: true,
-                  localIdentName: '[path]__[name]___[local]',
+                  localIdentName: '[name]-[local]',
```

Comment:
This would allow clashes in names. Its unlikely, but could happen, so add it as a possibility?

Having difference between the final output and dev, can also cause friction as it makes it much harder to look at the production site and find the issue when running in dev mode because all your class names are unrelated.

Reply:
Agree I would leave it as is